x-pack/plugins/maps/public/shared/layers/sources/es_search_source/es_search_source.js

```diff
@@ -136,7 +201,26 @@ export class ESSearchSource extends AbstractESSource {
     // By setting "fields", SearchSource removes all of defaults
     searchSource.setField('fields', searchFilters.fieldNames);
 
-    let featureCollection;
+    const resp = await this._runEsQuery(layerName, searchSource, 'Elasticsearch document request');
+
+    return {
+      hits: resp.hits.hits,
+      meta: {
+        areResultsTrimmed: resp.hits.total > resp.hits.hits.length
+      }
+    };
+  }
+
+  _isTopHits() {
+    const { useTopHits, topHitsSplitField, topHitsTimeField } = this._descriptor;
+    return useTopHits && topHitsSplitField && topHitsTimeField;
```

Comment:
Since were treating this as a boolean function, should probably return a proper boolean. Maybe just update the return value to `!!(useTopHits && topHitsSplitField && topHitsTimeField)`